src/github/entities/allowed-organization.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { OrganizationStatus } from 'src/common/constants';
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity()
+export class AllowedOrganization extends Base {
+  @Column({ type: 'varchar', length: 255, default: 'github' })
+  host: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true })
+  organizationName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminUserName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminEmail: string;
+
+  @Column('text', { array: true, default: [] })
+  activeUsers: string[];
+
+  @Column('text', { array: true, default: [] })
+  inactiveUsers: string[];
+
+  @Column({
+    type: 'enum',
+    enum: OrganizationStatus,
+    default: OrganizationStatus.ACTIVE,
+  })
+  status: OrganizationStatus;
+
+  @Column({ type: 'boolean', default: true })
+  allowAllUsers: boolean;
+}
```

Comment:
PullProofAI

This new entity `AllowedOrganization` looks well-defined with various properties related to an organization. However, consider adding validation logic for fields like `organizationName`, `adminUserName`, and `adminEmail` to ensure data integrity. Additionally, you can improve readability by organizing the columns in a more structured way.

```diff
+ // Consider adding validation decorators for fields like organizationName, adminUserName, and adminEmail
+ // Organize columns for better readability
```



<!-- This is an auto-generated comment by PullProofAI -->